src/style.css

```diff
@@ -1,3 +1,69 @@
-body {
+html {
+  font-family: "roboto", sans-serif;
+  font-size: 12px;
+  font-weight: 500;
+  --active-color: #00acdc;
```

Comment:
good practice to declare variables inside the [:root](https://www.w3schools.com/css/css3_variables.asp) selector
```css
:root {
    --active-color: #00acdc;
}
```